stdlib/public/core/Integers.swift.gyb

```diff
@@ -1678,7 +1678,82 @@ ${operatorComment(x.nonMaskingOperator, False)}
 
 }
 
-// Strideable conformance
+
+//===----------------------------------------------------------------------===//
+//===--- CustomStringConvertible conformance ------------------------------===//
+//===----------------------------------------------------------------------===//
+
+extension BinaryInteger {
+  @_inlineable // FIXME(sil-serialize-all)
+  @_versioned
+  @_transparent
+  internal func _description(
+    radix: Int = 10, uppercase: Bool = false
```

Comment:
I think it would be better to leave out the default values on this internal function, so we can't accidentally forget to pass on the values from the public API.